core/lib/Cardano/Address.hs

```diff
@@ -64,7 +61,10 @@ import GHC.Stack
 import Numeric.Natural
     ( Natural )
 
-import qualified Cardano.Codec.Bech32.Prefixes as CIP5
+import Cardano.Codec.Bech32
+    ( FromBech32 (..), ToBech32 (..), bech32With, fromBech32With )
+import Cardano.Codec.Bech32.Prefixes
+    ( addr, addr_vkh, stake_vkh )
```

Comment:
Please use a qualified import for this one to give some context to various hrp and remain consistent with other usages of that module.